tests/functional/firefox/test_new.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.new import FirefoxNewPage
+
+
+# We don't currently show the download button for up-to-date Firefox users.
+# TODO explore how could we run this test using a known out-of-date Firefox?
+@pytest.mark.skip_if_capabilities_include({'browserName': 'firefox'})
```

Comment:
After further discussion, we're going to go ahead with custom markers for the various conditions as we identify them. Here's a gist of a working patch based on this PR: https://gist.github.com/davehunt/566957e5cac7a40352bb